plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -345,17 +345,6 @@ public PostgreSqlClient(
                         .build());
     }
 
-    @Override
-    protected void dropSchema(ConnectorSession session, Connection connection, String remoteSchemaName, boolean cascade)
-            throws SQLException
-    {
-        if (cascade) {
-            // Dropping schema with cascade option may lead to other metadata listing operations. Disable until finding the solution.
```

Comment:
Stress test on CI passed in https://github.com/trinodb/trino/actions/runs/5851247174?pr=18663

Running `testDropSchemaCascade` and `testSelectInformationSchemaColumns` together with 100 invocationCount & 2 threadPoolSize also passed in my laptop.